spec/spec_helper.rb

```diff
@@ -1,63 +1,19 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
+$LOAD_PATH.unshift File.expand_path('../support', __FILE__)
```

Comment:
Oh, just realized, since we only support 2.1 and 2.2, we could use `__dir__` here and trim one `../` from each. YMMV.